flash/image/embedding/model.py

```diff
@@ -57,31 +62,38 @@ class ImageEmbedder(AdapterTask):
 
     training_strategies: FlashRegistry = IMAGE_EMBEDDER_STRATEGIES
     backbones: FlashRegistry = IMAGE_EMBEDDER_BACKBONES
+    transforms: FlashRegistry = IMAGE_EMBEDDER_TRANSFORMS
 
     required_extras: str = "image"
 
     def __init__(
         self,
         training_strategy: str,
-        embedding_dim: int = 128,
+        head: str,
+        pretraining_transform: str,
```

Comment:
Can this be inferred from the training strategy name? Or can you mix and match the transform recipes?

Reply:
you can mix and match, and even customize these recipes, we can add the logic for inferring these in the next PR